doc/source/reference/c-api/array.rst

```diff
@@ -1578,7 +1578,7 @@ Flag checking
 ^^^^^^^^^^^^^
 
 For all of these macros *arr* must be an instance of a (subclass of)
-:c:data:`PyArray_Type`, but no checking is done.
```

Comment:
Is this change indicating that checking _is_ now done?

Reply:
Dunno, I think we read must as the user having to take care of it, so it seemed a bit duplicate. But maybe it is not bad to repeat it.